src/plugins/jqplot.enhancedLegendRenderer.js

```diff
@@ -267,24 +268,28 @@
             }
 
             else {
+                sidx1 = sidx;
+                if(reverse) {
+                    sidx1 = plot.series.length - 1 - sidx;
```

Comment:
Why not use array.prototype.reverse() here?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reverse?v=control